sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Definitions/BepuAndStrideExtensions.cs

```diff
@@ -4,51 +4,31 @@
 using NQuaternion = System.Numerics.Quaternion;
 using SQuaternion = Stride.Core.Mathematics.Quaternion;
 
-using NRigidPose = BepuPhysics.RigidPose;
+using BRigidPose = BepuPhysics.RigidPose;
 using SRigidPose = Stride.BepuPhysics.Definitions.RigidPose;
 
-using NBodyVelocity = BepuPhysics.BodyVelocity;
+using BBodyVelocity = BepuPhysics.BodyVelocity;
 using SBodyVelocity = Stride.BepuPhysics.Definitions.BodyVelocity;
 
 using System.Runtime.CompilerServices;
 
 
 internal static class BepuAndStrideExtensions
```

Comment:
Since these are dealing with System.Numerics and Stride.Math should this be added directly to the Stride math library? I feel like these would be useful in many places not only Bepu.

Reply:
Some of these seem to already exist in the 3d importer as well https://github.com/stride3d/stride/blob/0e053a3b8873f49e9e0e5450a3dfad368a6ed042/sources/tools/Stride.Importer.3D/Utils.cs#L13

Reply:
Sure, feel free to open a PR for that :P